content/en/docs/ops/ambient/install/istioctl-installation/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Install with Istioctl
+description: Install Istio in Ambient mode with Istioctl.
+weight: 4
+owner: istio/wg-environments-maintainers
+test: yes
+---
+
+This guide shows you how to install Istio in ambient mode with Istioctl.
+Aside from following the demo in [Getting Started with Ambient Mode](/docs/ops/ambient/getting-started/),
```

Comment:
we recommend people use Helm on the Helm page, so maybe we should point that out here too?

Maybe a callout box with the "The recommended installation method for Istio in ambient mode is Helm." rather than this

Reply:
Done.